README.md

```diff
@@ -66,8 +66,8 @@ In order to run this project you need:
 Clone this repository to your desired folder:
 
 ```sh
-  cd awesome-bokks-with-es6
-  git clone git@github.com:noelfoka/awesome-bokks-with-es6.git
+  cd ato-do-list
+  git clone https://github.com/noelfoka/to-do-list.git
 ```
```

Comment:
- [ **OPTIONAL** ] Good job for adding all the changes mentioned by the previous reviewer. However, I think it would be better if you improve your Getting Started section with other information needed by users such how to `access` the cloned directory, to `install` dependencies, to `run build` for production, `run/use` your application,... To help you, I suggest you refer **[this article.](https://gist.github.com/tresorsawasawa/cc5e9a9e5dd20c0179ae98eca3b1ae96)**